plugins/interactive.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: interactive
+spec:
+  version: v1.0.3
+  homepage: https://github.com/similarweb/kubectl-interactive
+  shortDescription: A kubectl plugin that provides interactive, easy-to-use and flexible interface
+  description: |
+    A kubectl plugin that is easy-to-use, flexible, and interactive
+    You will have the ability to roam around the Kubernetes world with a blazing fast selection menu
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/similarweb/kubectl-interactive/releases/download/v1.0.3/kubectl-interactive_1.0.3_Darwin_x86_64.tar.gz
+    sha256: d11624612a8924fa61b6e14e7d1d48731e283bbf85e2105dbc56ff07b44441a0
+    files:
+    - from: "*"
+      to: "."
```

Comment:
this is the default, you can remove `files:` altogether.

Reply:
removed